src/improvements/tasks/sep/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,706 @@
+## Understanding Task Calldata
+
+Multicall3DelegateCall calldata:
```

Comment:
How will this be validated? Can signers just copy it and ctrl+f for this string in the tenderly sim?

We should also include the target address for `Multicall3DelegateCall` as that must not change. 

I'm out of time now, but in the morning I'll generate a quick script to compute the calldata directly from a config to facilitate this review.